bin/katello-certs-check

```diff
@@ -151,12 +151,15 @@ function check-priv-key () {
 
 function check-ca-bundle () {
     printf "Checking CA bundle against the certificate file: "
+    ERROR_PATTERN="error [0-9]+ at"
     CHECK=$(openssl verify -CAfile $CA_BUNDLE_FILE -purpose sslserver -verbose $CERT_FILE 2>&1)
-    if [[ $? == "0" ]]; then
-        success
+    CHECK_STATUS=$?
```

Comment:
@ekohl Does this work to make it explicit and reduce the risk you are worried about?

Reply:
That's another way that reduces the risk.